docs/security.html

```diff
@@ -20,7 +20,8 @@
     In release 0.9.0.0, the Kafka community added a number of features that, used either separately or together, increases security in a Kafka cluster. The following security measures are currently supported:
     <ol>
         <li>Authentication of connections to brokers from clients (producers and consumers), other brokers and tools, using either SSL or SASL (Kerberos).
-        SASL/PLAIN can also be used from release 0.10.0.0 onwards.</li>
+        SASL/PLAIN can be used from release 0.10.0.0 onwards. SASL/SCRAM-SHA-256 and SASL/SCRAM-SHA-512 can be
```

Comment:
The phrasing is a bit awkward - "We support SASL (Kerberos), and then we added SASL/PLAIN and then we added SASL/SCRAM". Almost makes it seem like the additional protocols are an afterthought and may deter usage.

Perhaps just something like this will be clearer:

We support the following mechanisms: 
* SASL/KERBEROS - starting at version 0.9.0.0
* SASL/PLAIN - starting at version 0.10.0.0
* SASL/SCRAM-SHA-256 and SASL/SCRAM-SHA-512 - starting at version 0.10.2.0